EaseProbe Benchmark Test Report (#161)
megaease/easeprobe: docs/Benchmark.md

      session required pam_limits.so
      ```

> - **Build**: the following PRs are important for this test, must be included:

Reviewer: ```suggestion
- **Build**: the following PRs are needed for this test and we must ensure they are included (already merged in `main`):
```

- with 5s, 10s, 30s, 60s probe interval
- With 5 mins running, and check the CPU, Memory, and Network connection.

Using the the following command to observe the performance:
- `top` and `htop` to check the CPU and Memory usage.
- `netstat  -nat  |  awk  '{print  $6}'  |  sort  |  uniq  -c  |  sort  -n` to check the Network connection.

## 3. Test Data

> The test data is come from the top 1M websites (https://majestic.com/reports/majestic-million)

Reviewer: ```suggestion
The test data is taken from the top 1M websites (https://majestic.com/reports/majestic-million)
```

> You have these test data by visit this document - [EaseProbe Test](../../resources/test/)

Reviewer: ```suggestion
The test data can be found under the [EaseProbe Test](../../resources/test/) folder.
```
